planning/behavior_path_planner/src/scene_module/avoidance_by_lc/module.cpp

```diff
@@ -0,0 +1,617 @@
+// Copyright 2023 TIER IV, Inc.
```

Comment:
This file is no longer needed. Removed by https://github.com/autowarefoundation/autoware.universe/pull/3632.

Reply:
Thank you.
Done. https://github.com/autowarefoundation/autoware.universe/pull/3604/commits/b23f68ee65f42c05837a96effc3646839d6a5c79